addon/components/power-select/options.js

```diff
@@ -16,8 +16,9 @@ export default Ember.Component.extend({
     if (this.get('role') === 'group') { return; }
 
     this.element.addEventListener('mouseup', e => {
-      if (e.target.dataset.optionIndex) {
-        this.get('select.actions.choose')(this._optionFromIndex(e.target.dataset.optionIndex), e);
+      const optionIndex = Ember.$(e.target).closest('li').data('option-index').toString();
```

Comment:
I prefer this in case the custom html inside the options contains (God knows why) more nested `<li>`s elements.